src/sentry/api/event_search.py

```diff
@@ -683,7 +686,7 @@ def generic_visit(self, node, children):
         return children or node
 
 
-def parse_search_query(query, allow_boolean=True):
+def parse_search_query(query, allow_boolean=True, exact_timestamp=False):
```

Comment:
Starting to think we should have a different entry point function that does the thing we want instead of having a buffet of modal flags that we generally turn all on or leave off.

Reply:
Yeah I'm thinking about it. As more and more options are added, this can get messy quickly. Especially because it can change existing behavior elsewhere in the product as I mentioned above.

Reply:
Yes, a separate entrypoint would allow us to ensure we don't break existing behavior with more confidence.